api/src/main/kotlin/com/gmtkgamejam/models/Skills.kt

```diff
@@ -0,0 +1,33 @@
+package com.gmtkgamejam.models
+
+import kotlin.math.pow
+
+enum class Skills(val bitwiseId: Int) {
+    `2DArt`(1),
```

Comment:
Small point: usually enum values are named in all caps.

Reply:
Ah, thanks for the spot - can't believe IDEA didn't tell me off for that!